cpp/src/strings/strings_column_view.cu

```diff
@@ -113,15 +110,14 @@ void print( strings_column_view strings,
     // copy strings into output buffer
```

Comment:
Would it be worth removing this function?  With the fix to create_offsets(),  cudf::util::print() should handle strings directly (I think the formatting is slightly different though).

Reply:
> Would it be worth removing this function? With the fix to create_offsets(), cudf::util::print() should handle strings directly (I think the formatting is slightly different though).

I'm not seeing where `cudf::util::print()` is defined.